### PR TITLE
docs: update links

### PR DESCRIPTION
**Pull request by ManfredHu**

### CODE REVIEW

File: content/docs/hooks-reference.md [hunk 1]
```diff
@@ -528,20 +528,21 @@ useDebugValue(date, date => date.toDateString());
 const [deferredValue] = useDeferredValue(value);
 ```
 
-`useDeferredValue` accepts a value and returns a new copy of the value that will defer to more urgent updates. If the current render is the result of an urgent update, like user input, React will return the previous value and then render the new value after the urgent render has completed.
+`useDeferredValue` 接收一个值并返回接收值的拷贝副本，该副本将推迟到更紧急的更新。如果当前呈现是紧急更新的结果(如用户输入)，那么React将返回先前的值，然后在紧急呈现完成后呈现新值。
```
Comment (awxiaoxian2020) [inline content/docs/hooks-reference.md:531]:
```suggestion
`useDeferredValue` 接收一个值并返回接收值的拷贝副本，该副本将推迟到更紧急的更新。如果当前呈现是紧急更新的结果（如用户输入），那么 React 将返回先前的值，然后在紧急呈现完成后呈现新值。
```

File: content/docs/hooks-reference.md [hunk 2]
```diff
@@ -528,20 +528,21 @@ useDebugValue(date, date => date.toDateString());
...
 
-This hook is similar to user-space hooks which use debouncing or throttling to defer updates. The benefits to using `useDeferredValue` is that React will work on the update as soon as other work finishes (instead of waiting for an arbitrary amount of time), and like [`startTransition`](/docs/react-api.html#starttransition), deferred values can suspend without triggering an unexpected fallback for existing content.
+这个hook类似user-space钩子，被用来做防抖或节流延迟更新。
+
+使用 `useDeferredValue` 的好处是，React将在其他工作完成后立即进行更新(而不是等待任意的一段时间),如同 [`startTransition`](/docs/react-api.html#starttransition)，deferred values可以挂起，而不会触发原有内容的意外回退。
 
 #### Memoizing deferred children {#memoizing-deferred-children}
-`useDeferredValue` only defers the value that you pass to it. If you want to prevent a child component from re-rendering during an urgent update, you must also memoize that component with [`React.memo`](/docs/react-api.html#reactmemo) or [`React.useMemo`](/docs/hooks-reference.html#usememo):
+`useDeferredValue` 只会推迟您传递给它的值。如果要防止子组件在紧急更新期间重新呈现，还必须使用 memoize 的组件比如  [`React.memo`](/docs/react-api.html#reactmemo) 或者 [`React.useMemo`](/docs/hooks-reference.html#usememo):
```
Comment (awxiaoxian2020) [inline content/docs/hooks-reference.md:538]:
```suggestion
`useDeferredValue` 只会推迟传递给它的值。如果要防止子组件在紧急更新期间重新呈现，还必须使用 memoize 的组件比如  [`React.memo`](/docs/react-api.html#reactmemo) 或者 [`React.useMemo`](/docs/hooks-reference.html#usememo):
```

File: content/docs/hooks-reference.md [hunk 3]
```diff
@@ -558,25 +559,26 @@ function Typeahead() {
...
+
 
 ### `useTransition` {#usetransition}
 
 ```js
 const [isPending, startTransition] = useTransition();
 ```
 
-Returns a stateful value for the pending state of the transition, and a function to start it.
+返回判断pending的状态值，并返回启动转换的函数。
```
Comment (awxiaoxian2020) [inline content/docs/hooks-reference.md:571]:
```suggestion
返回判断 pending 的状态值，并返回启动转换的函数。
```

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

**Review by daochouwangu — DISMISSED** [review]

**Comment by awxiaoxian2020** [comment]
@ManfredHu Could you please fix the conflicts?

**Review by awxiaoxian2020 — APPROVED** [review]

**Review by daochouwangu — APPROVED** [review]

**Pull request merged**